docs/migration_guide.rst

```diff
@@ -1,6 +1,63 @@
 Migration guide
 ===============
 
+*********************
+0.X.X Migration guide
+*********************
+
+0.X.X makes the first step to remove the Cairo lang dependency!
+
+Some classes/functions from the Cairo lang are rewritten and are the part of our library:
+
+- :ref:`transactions' dataclasses <Transaction's dataclasses>`
+- `get_selector_from_name` and `get_storage_var_address` functions
+
+Deprecation
+-----------
+
+`compute_invoke_hash` is deprecated in favour of `compute_transaction_hash`.
+
+Breaking changes
+----------------
+
+1. `InvokeFunction` is replaced by the `Invoke` dataclass (behaviour is the same, just the name is changed).
```

Comment:
I think double backticks should be used here and in other places ` ``Invoke``  `. Current version italicizes the text instead.